tests/Application/Persistence/RetrySafeEntityManagerTest.php

```diff
@@ -152,6 +152,65 @@ public function testFlushWithRetry(): void
         $this->retrySafeEntityManager->flush();
     }
 
+    public function testRefresh(): void
```

Comment:
IMHO tests get easier to read if we use full sentance names, e.g. `testRefreshCallsUnderlyingRefresh`

But honestly the code under test here seem so simple I'm not sure it's really worth having a unit test for it - what plausible mistakes would the unit test help us catch that won't be caught just as well by existing integration tests and/or Psalm? This isn't testing the case where the underlying throws, so it's really just testing:

```
    public function refresh($object, ?int $lockMode = null): void
    {
            $this->entityManager->refresh($object, $lockMode);
     }

Reply:
I think it's still better to have a simple "sanity check" test than no test, as this also protects against future changes to the code under test.